recipes/castorapi/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "castorapi" %}
+{% set version = "0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4544ab11e1144c1a66a466f5ffc379ff1031babe6d74c0fba027f3d4fd6d3b0c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - json5
+    - pandas
+    - pip
+    - progressbar2
+    - python
+    - requests
+  run:
+    - json5
+    - pandas
+    - progressbar2
+    - python
+    - requests
+
+test:
+  imports:
+    - castorapi
+
+about:
+  home: "https://github.com/wouterpotters/castorapi"
+  license: MIT
+  license_family: MIT
+  license_file: "LICENSE"
+  summary: "Python API wrapper for Castor EDC to fetch data from you clinical study."
+  doc_url: "https://github.com/wouterpotters/castorapi/README.md"
```

Comment:
The `doc_url` is not found. You can probably use `https://github.com/wouterpotters/castorapi#usage` instead.

Reply:
fixed